<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VersionManager</class>
 <widget class="QWidget" name="VersionManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>版本管理</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>980</width>
        <height>598</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="0">
        <widget class="QGroupBox" name="gb_modmanage">
         <property name="title">
          <string>Mod管理</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="1">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="pb_openmodfoder">
            <property name="text">
             <string>打开mod文件夹</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <layout class="QGridLayout" name="gl_modlist"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>基础</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="2" column="2" colspan="2">
           <widget class="QLabel" name="l_forgeversion">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QPushButton" name="pb_del">
            <property name="text">
             <string>删除</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLabel" name="l_optifineversion">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>版本</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" rowspan="5">
           <widget class="QLabel" name="l_icon">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="2" colspan="2">
           <widget class="QLineEdit" name="le_name"/>
          </item>
          <item row="5" column="3">
           <widget class="QPushButton" name="pb_openfoder">
            <property name="text">
             <string>打开版本文件夹</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Fabric版本</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Forge版本</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Optifine版本</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>名称</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="l_fabricversion">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="pb_reinstall">
            <property name="text">
             <string>重新安装</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2" colspan="2">
           <widget class="QLabel" name="l_version">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QPushButton" name="pb_changeicon">
            <property name="text">
             <string>更改图标</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>设置</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5"/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
